### ISSUE TITLE
nan loss

### ISSUE THREAD

**Issue by tanshuai0219** [a-r-r-o-w/finetrainers#143]
### System Info / 系統信息

CUDA:  12.4  
python: 3.10
diffusers: 0.33.0.dev0

### Information / 问题信息

- [ ] The official example scripts / 官方的示例脚本
- [X] My own modified scripts / 我自己修改的脚本和任务

### Reproduction / 复现过程

1. I randomly selected a few videos to fine-tune hunyuanvideo
2. But I ended up with nan loss at the first step.
3. the log is 
`Training steps:   2%|██                                                                                                        | 1/50 [00:56<46:19, 56.73s/it, loss=nan, lr=2e-7]`

### Expected behavior / 期待表现

Get the correct loss value.

**Comment by a-r-r-o-w** [comment]
@tanshuai0219 I believe we interacted in the Diffusers issue too. Upgrading pytorch to 2.5.1 has been confirmed to fix the nan training loss problem by atleast 3 other people now. Could you give it a try?

**Comment by tanshuai0219** [comment]
> @tanshuai0219 I believe we interacted in the Diffusers issue too. Upgrading pytorch to 2.5.1 has been confirmed to fix the nan training loss problem by atleast 3 other people now. Could you give it a try?

Yes,it works for me~ The loss value is normal now. Thanks!

**Comment by BlackTea-c** [comment]
same problem.butmy torch version is 2.5.1
